autopilot/sagemaker_autopilot_abalone_parquet_input.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
<p>It seems like this notebook stops halfway through. Once the training is completed, showcase and interpret the results, what has been accomplished, and how customers could adapt or extend this for other use cases. The notebooks are not purely for running code; they should also educate customers and add context around the purpose and benefit of these features. I see that the notebook you're using as a starting point doesn't do a great job of this either. </p><p> </p><p>Take a look at some other Autopilot notebooks to get a sense of how you can conclude the notebook (although no need to specifically perform inference or deploy an endpoint if that's out of scope of this example): </p><p> </p><p><a href="https://github.com/aws/amazon-sagemaker-examples/blob/master/autopilot/sagemaker_autopilot_direct_marketing.ipynb" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/master/autopilot/sagemaker_autopilot_direct_marketing.ipynb</a></p><p><a href="https://github.com/aws/amazon-sagemaker-examples/blob/master/autopilot/autopilot_customer_churn_high_level_with_evaluation.ipynb" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/master/autopilot/autopilot_customer_churn_high_level_with_evaluation.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3093/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>Can we refer to these two notebooks for the next steps such as deploying the best model and running inference?</p>

Reply:
I linked to those as more general references for how results are shown and explained. I'm not sure if they are the most relevant links to share, but feel free to add links if you do think it's helpful.

I think the essential part is: you end the notebook with a 50-line code block. Please add more text explaining what the results and impact of this is. How can the output artifacts be interpreted, etc? This will be a more smooth and satisfying end for customers who are following along with the notebook.

Reply:
I think they are helpful to link at the end as all these things except the job submission with a parquet data has been covered in other notebooks. I am not sure what else can be added here without duplicating what is already there in other notebooks. In my opinion, as long as we give good pointers to the customers we should be okay as this is a getting started notebook.